extensions/tls-registry/spi/src/main/java/io/quarkus/tls/TlsConfiguration.java

```diff
@@ -102,7 +102,12 @@ static Optional<TlsConfiguration> from(TlsConfigurationRegistry registry, Option
 
     /**
      * Returns the name which was associated with this configuration
+     * <p>
+     * Note: Although this was made default in order to not break deep integrations, it is strongly that the method
+     * be implemented.
      */
-    String getName();
+    default String getName() {
+        return "unset";
```

Comment:
Wouldn't a unique default name be better - something like 
```suggestion
        return "tls-configutation-" + System.identityHashCode(this);
```

Reply:
I don't see why it would be